src/websocket-client.ts

```diff
@@ -67,6 +69,8 @@ interface WebsocketClientEvents {
   update: (response: WsDataEvent & WsKeyObject) => void;
   /** Exception from ws client OR custom listeners */
   error: (response: any) => void;
+  /** Information update */
+  channelConnInfo: (response: WsChannelConnInfoEvent) => void;
```

Comment:
I'm cautious of adding new event categories. Rather than emitting a new event category, thoughts on including this either in the `response` category (the req/rep category where sub/unsub confirmations are also sent), or the `update` category? 

Seems to me the `response` category would be a better fit, since this is a reaction to a request from the client (similar to a REST API response).

Reply:
Done as you wish, I just want to get rid of those Unhandled errors in console :)

Reply:
Fair enough, I really appreciate the PR :)